packages/manager/modules/telecom-dashboard/src/index.js

```diff
@@ -18,15 +19,20 @@ import 'ovh-manager-webfont/dist/css/ovh-font.css';
 import './telecom-dashboard.scss';
 import './telecom-dashboard.less';
 
+import IdentityCheckForm from './identity-check/form';
+
 import dashboardCtrl from './telecom-dashboard.controller';
 import billsCtrl from './bills/telecom-dashboard-bills.controller';
 import guidesCtrl from './guides/telecom-dashboard-guides.controller';
+import IdentityCheckMessageCtrl from './identity-check/message/telecom-dashboard-identity-check-message.controller';
 
 import template from './telecom-dashboard.html';
 import billsTemplate from './bills/telecom-dashboard-bills.html';
 import guidesTemplate from './guides/telecom-dashboard-guides.html';
+import identityCheckMessageTemplate from './identity-check/message/telecom-dashboard-identity-check-message.html';
 
 import billsService from './bills/telecom-dashboard-bills.service';
+import identityCheckService from './identity-check/telecom-dashboard-identity-check.service';
```

Comment:
I know it's was here before, but i think it's better to group imports by component not by types.
I think it's better and more relevant.

```suggestion
import IdentityCheckForm from './identity-check/form';

import template from './telecom-dashboard.html';
import dashboardCtrl from './telecom-dashboard.controller';

import billsTemplate from './bills/telecom-dashboard-bills.html';
import billsCtrl from './bills/telecom-dashboard-bills.controller';
import billsService from './bills/telecom-dashboard-bills.service';

import guidesTemplate from './guides/telecom-dashboard-guides.html';
import guidesCtrl from './guides/telecom-dashboard-guides.controller';

import identityCheckMessageTemplate from './identity-check/message/telecom-dashboard-identity-check-message.html';
import IdentityCheckMessageCtrl from './identity-check/message/telecom-dashboard-identity-check-message.controller';

import identityCheckService from './identity-check/telecom-dashboard-identity-check.service';
```